### PR TITLE
fix(kre-py): jetstream subscription error

### PR DESCRIPTION
**Pull request by alexrodfe**
## Why
kre-py runner was not recovering correctly when it couldn't connect to jetstream stream.

## What
Make kre-py exit execution when not subscribing correctly so it tries subscription again.

### CODE REVIEW

File: kre-py/src/main.py [hunk 1]
```diff
@@ -81,17 +81,24 @@ async def execute_handler_init(self) -> None:
...
-            subject=self.config.nats_input,
-            queue=self.runner_name,
-            durable=self.runner_name,
-            cb=self.create_message_cb(),
-            config=ConsumerConfig(
-                deliver_policy=DeliverPolicy.NEW,
-            ),
-            manual_ack=True,
-        )
+        try:
```
Comment (alejandrogr) [inline kre-py/src/main.py:84]: Good catch! With this i presume that the pod is restarted, hence the nats subcription is retried?

Comment (alexrodfe) [reply kre-py/src/main.py:84]: Yes it is! The pod will restart and try again the subscription until it achieves it.

**Pull request merged**